explorer/models/taskNode.ts

```diff
@@ -1,21 +1,23 @@
 import * as ContainerModels from 'azure-arm-containerregistry/lib/models';
-import { ResourceManagementClient, SubscriptionClient, SubscriptionModels } from 'azure-arm-resource';
+import { SubscriptionModels } from 'azure-arm-resource';
 import * as opn from 'opn';
 import * as vscode from 'vscode';
-import { AzureAccount, AzureSession } from '../../typings/azure-account.api';
+import { AzureAccount } from '../../typings/azure-account.api';
 import * as acrTools from '../../utils/Azure/acrTools';
 import { AzureUtilityManager } from '../../utils/azureUtilityManager';
 import { NodeBase } from './nodeBase';
 
 /* Single TaskRootNode under each Repository. Labeled "Build Tasks" */
 export class TaskRootNode extends NodeBase {
+    public static readonly contextValue: string = 'taskRootNode';
+    private _onDidChangeTreeData: vscode.EventEmitter<NodeBase> = new vscode.EventEmitter<NodeBase>();
+    public readonly onDidChangeTreeData: vscode.Event<NodeBase> = this._onDidChangeTreeData.event;
     constructor(
         public readonly label: string,
-        public readonly contextValue: string,
         public subscription: SubscriptionModels.Subscription,
         public readonly azureAccount: AzureAccount,
         public registry: ContainerModels.Registry,
-        public readonly iconPath: any = {}
+        public readonly iconPath: any = {},
```

Comment:
The code in getTreeItem uses null for iconPath. I guess both `{}` or `null` work. Can you update them to be consistent?